code-challenges.test.js

```diff
@@ -24,25 +33,108 @@
 
 // b) Create the function that makes the test pass.
 
+//create fib variable
+//for fib define num values
+//for loop states that the function will act on numbers greater than 2, but less than or equal to the example inputs (6,10). Function will iterate through the given parameters
+//return array of 6 and 10 in fib.
+
+
+
+
```

Comment:
function based on syllabus

//the function name is `fib`
//use an array beginning with 1,1 just like it shows on the fibonacci sequence
// the length has to be reconsidered because our array begins with a length of 2 already for the fibonacci sequence [1, 1]. Therefore, we subtract 2 from the number so we can push the right amount of indexes to give us the expected length.
//use a different variable name to hold that array `fibArray`


const fib = (num) => {

let fibArray = [1, 1]

for(let i = 0; i < num - 2; i++) {
fibArray.push(fibArray[i] + fibArray[i+1]) 
}
return fibArray 
}